SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -105,61 +108,102 @@ bool WriteLoadbalancingSection(TSharedRef<TJsonWriter<>> Writer, const FName& Wo
 
 } // anonymous namespace
 
-void SetLevelEditorPlaySettingsWorkerTypes(const TMap<FName, FWorkerTypeLaunchSection>& InWorkers)
+void SetLevelEditorPlaySettingsWorkerType(const FWorkerTypeLaunchSection& InWorker)
 {
 	ULevelEditorPlaySettings* PlayInSettings = GetMutableDefault<ULevelEditorPlaySettings>();
 
-	PlayInSettings->WorkerTypesToLaunch.Empty(InWorkers.Num());
+	PlayInSettings->WorkerTypesToLaunch.Empty(1);
+
+	// TODO: Engine PR to remove PlayInSettings WorkerType map.
+	PlayInSettings->WorkerTypesToLaunch.Add(SpatialConstants::DefaultServerWorkerType, InWorker.NumEditorInstances);
+}
+
+uint32 GetWorkerCountFromWorldSettings(const UWorld& World)
+{
+	const ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(World.GetWorldSettings());
+
+	if (WorldSettings == nullptr)
+	{
+		UE_LOG(LogSpatialGDKDefaultLaunchConfigGenerator, Error, TEXT("Missing SpatialWorldSettings on map %s"), *World.GetMapName());
+		return 1;
+	}
 
-	if (InWorkers.Num() == 0)
+	if (WorldSettings->bEnableMultiWorker == false)
 	{
-		UE_LOG(LogSpatialGDKDefaultLaunchConfigGenerator, Warning, TEXT("No workers specified in SetLevelEditorPlaySettingsWorkerType."));
+		return 1;
 	}
 
-	for (const auto& Worker : InWorkers)
+	FSpatialGDKEditorModule& EditorModule = FModuleManager::GetModuleChecked<FSpatialGDKEditorModule>("SpatialGDKEditor");
+	uint32 NumWorkers = 0;
+	if (WorldSettings->DefaultLayerLoadBalanceStrategy == nullptr)
 	{
-		if (Worker.Value.bAutoNumEditorInstances)
```

Comment:
This check has been removed, either way, I think this function will have to be rewritten in a future PR, to make use of the new exposed information from the LB strategy (such as MinimumRequiredWorkers).